src/Command/BakeCommand.php

```diff
@@ -164,4 +164,28 @@ protected function deleteEmptyFile(string $path, ConsoleIo $io): void
             $io->out(sprintf('<success>Deleted</success> `%s`', $path), 1, ConsoleIo::NORMAL);
         }
     }
+
+    /**
+     * Check if a column name is valid
+     *
+     * The Regex used here basically states that:
+     * the column name has to start with a character (lower or upper case) or an underscore and
+     * further characters are allowed to be either lower or upper case characters, numbers or underscores
+     *
+     * Anything else like
```

Comment:
The valid charset is stated above so listing out various invalid ones against is redundant. It will unnecessarily make the API docs longer.